history.js

```diff
@@ -240,7 +240,7 @@
          */
         "state": {
             get: function() {
-                return stateStorage[windowLocation.href] || null;
+                return typeof stateStorage[windowLocation.href] === 'object' ? JSON.parse(JSON.stringify(stateStorage[windowLocation.href])) : (stateStorage[windowLocation.href] || null);
```

Comment:
Hi!
Write a few lines to get a better read. I would also like to see changes to the file `history.ielte7.js`
Thanks!